sdk/core/azure-core/azure/core/_pipeline_client_async.py

```diff
@@ -168,3 +173,52 @@ def _build_pipeline(self, config, **kwargs): # pylint: disable=no-self-use
             transport = AioHttpTransport(**kwargs)
 
         return AsyncPipeline(transport, policies)
+
+    async def _make_pipeline_call(self, request, stream, **kwargs):
```

Comment:
Nitpick; I'd make `stream` a keyword-only argument.

Reply:
sounds good. I've left it as a keyword only argument on send_request